src/main/java/de/gebit/plugins/autoconfig/handlers/java/JavaHandler.java

```diff
@@ -11,12 +11,12 @@
 import com.intellij.compiler.CompilerConfiguration;
 import com.intellij.compiler.CompilerConfigurationImpl;
 import com.intellij.compiler.impl.javaCompiler.BackendCompiler;
+import com.intellij.debugger.settings.DebuggerSettings;
 import com.intellij.openapi.project.Project;
 import de.gebit.plugins.autoconfig.UpdateHandler;
 import de.gebit.plugins.autoconfig.handlers.AbstractHandler;
-import de.gebit.plugins.autoconfig.model.AnnotationProcessor;
 import de.gebit.plugins.autoconfig.model.Compiler;
-import de.gebit.plugins.autoconfig.model.JavaConfiguration;
+import de.gebit.plugins.autoconfig.model.*;
```

Comment:
Why is a star-import used here? Do we have different import-optimizer settings in use?